src/components/common/PersonalInfoInput.tsx

```diff
@@ -40,14 +76,21 @@ function PersonalInfoInput(props: PersonalInfoInputProps) {
         )}
       </StyledTitleWrapper>
       <StyledInputWrapper>
-        <input type={inputInfo.type} {...inputValue} placeholder={inputInfo.placeholder} />
-        <SignupOption type={inputType} error={isError} />
+        <input
+          type={inputInfo.type}
+          {...inputValue}
+          placeholder={inputInfo.placeholder}
+          disabled={inputType === 'emailConfirm' && isCompleteList[inputType]}
+        />
+        <SignupOption
+          type={inputType}
+          error={isError}
+          isCompleteList={isCompleteList}
+          value={inputValue.value}
+          handleConfirm={handleConfirm}
+        />
       </StyledInputWrapper>
-      {(inputType === 'passwordConfirm' && passwordError) || (isError && inputValue.value) ? (
-        <StyledNoticeErr>{inputInfo.notice}</StyledNoticeErr>
-      ) : (
-        <StyledNoticeErr />
-      )}
+      {getNotice(inputType, isConfirm)}
```

Comment:
```suggestion
      <StyledNoticeErr>{getNotice(inputType, isConfirm)}</StyeldNoticeErr>
```

이렇게하고 getNotice 함수에서 notice 텍스트만 넘겨주는 방식은 어떨까유